base/applications/clipbrd/lang/pt-PT.rc

```diff
@@ -0,0 +1,67 @@
+/* Translation and UTF-8 Conversion by Gabriel Aguiar(fgsoftware1) <fgygh5804@gmail.com,fgsoftwarestudio@gmail.com> (July, 2021) */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```

Comment:
Did you test this?

Reply:
no

Reply:
i dont have reactos but im trying compile as i say above

Reply:
> i dont have reactos but im trying compile as i say above

You did not mention that anywhere.